src/timings.txt

```diff
@@ -0,0 +1,53 @@
+Part A
```

Comment:
Remember we talked about going first yourself on the PR?
It will always happen..

Reply:
I walk through it but missed this file :\